Makefile

```diff
@@ -1,6 +1,7 @@
-.PHONY: clean install analysis test test-install develop docs docs-install
+.PHONY: clean venv install analysis test test-install develop docs docs-install
```

Comment:
venv isn't, but moving the Python requirement check into the venv target means that it should be PHONY if we want to run the check every time we run `make install`, not just the first that runs `venv`.